go/interpreter/lfvm/opcode_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package lfvm
+
+import (
+	"slices"
+	"testing"
+)
+
+func TestOpcode_String(t *testing.T) {
+	tests := []struct {
+		name   string
+		opcode OpCode
+	}{
```

Comment:
Because every print returns a new string, I am afraid we should test all of them.
(the problem being here that someone misspelled an opcode)
Can we do this with reflection?